packages/x-data-grid/src/models/gridExport.ts

```diff
@@ -40,6 +40,12 @@ export interface GridFileExportOptions<Api extends GridApiCommon = GridApiCommun
    * @returns {GridRowId[]} The list of row ids to export.
    */
   getRowsToExport?: (params: GridGetRowsToExportParams<Api>) => GridRowId[];
+  /**
```

Comment:
We should probably document more why `false` is the default if it is recommended to set this to `true` :+1: 
Will the default become `true` in the next major?
If not, explain what is preventing it in the JSDoc.

If we don't explain our reasonning, it looks like we are not securing our library by default.

Reply:
Thanks for looking into it!
I'm leaning towards setting it to `true` by default, but I wanted to discuss this with the team first as it's technically a breaking change 👍🏻